manual_deployment/global-args-security.groovy

```diff
@@ -60,10 +60,6 @@ if ( zafiraServiceUrl != null && !envVars.containsKey("ZAFIRA_SERVICE_URL") ) {
   envVars.put("ZAFIRA_SERVICE_URL", zafiraServiceUrl)
```

Comment:
ZAFIRA_SERVICE_URL should be removed as well. We did it several releases ago and forgot to clean